core/src/epicli/data/azure/defaults/epiphany-cluster.yml

```diff
@@ -0,0 +1,96 @@
+kind: epiphany-cluster
+title: "Epiphany cluster Config"
+provider: azure
+name: "default"
+specification:
+  prefix: default
+  name: epiphanycluster
+  admin_user:
+    name: operations # YOUR-ADMIN-USERNAME
+    key_path: /root/.ssh/epiphany-operations/id_rsa # YOUR-SSH-KEY-PATH
+  cloud:
+    k8s_as_cloud_service: False
+    subscription_name: YOUR-SUB-NAME
+    vnet_address_pool: 10.1.0.0/20
+    use_public_ips: False # When not using public IPs you have to provide connectivity via private IPs (VPN)
+    use_service_principal: False
```

Comment:
use_service_principal is Azure only so it can be removed from the AWS config if present.